server/src/main/java/com/linecorp/centraldogma/server/internal/storage/DirectoryBasedStorageManager.java

```diff
@@ -228,7 +243,7 @@ private T create0(Author author, String name, long creationTimeMillis) {
                 try {
                     Util.deleteFileTree(f);
                 } catch (IOException e) {
-                    logger.warn("Failed to delete a partially created project: {}", f);
+                    logger.warn("Failed to delete a partially created project: " + f, e);
```

Comment:
`{}", f, e`?

Reply:
IIUC, slf4j does support `logger.warn(format, arg, throwable)` api, so I chose [`warn​(String msg, Throwable t)`](https://www.slf4j.org/api/org/slf4j/Logger.html#warn(java.lang.String,java.lang.Throwable)).

https://www.slf4j.org/api/org/slf4j/Logger.html
FYI
<img src="https://user-images.githubusercontent.com/1866157/61602976-be47b680-ac76-11e9-952c-0598c045ca2d.png" width=600>

Reply:
If you try, you'll realize it will just work. If the last argument in `Object...` is a `Throwable` and it has no matching `{}`, its stack trace will be logged.

Reply:
Ahh............ I didn't know that. I found the code which extracts throwable from the arguments.

```java
final Object lastEntry = argArray[argArray.length - 1];
if (lastEntry instanceof Throwable) {
    return (Throwable) lastEntry;
}
```
https://github.com/qos-ch/logback/blob/master/logback-classic/src/main/java/ch/qos/logback/classic/spi/EventArgUtil.java#L23-L25

Thanks for kind advice. 🙏

Reply:
While seeing the `logback` codes, I thought it seems to always extract the last argument exception without matching `{}`.

I tried some case:
```java
logger.warn("hello {}", new Exception());
// [WARN] hello {}                    <- ignore matching and print stack trace
// java.lang.Exception: null
logger.warn("hello {}", "world", new Exception());
// [WARN] hello world                 
// java.lang.Exception: null
logger.warn("hello {}", new Exception(), "world");
// [WARN] hello java.lang.Exception    <- {} matchs with Exception and no stack trace
```